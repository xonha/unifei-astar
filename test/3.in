9
000000000
F00000000
000000000
000T00T00
000000000
000T0T0FF
00T0T000T
000000TT0
F00000000
