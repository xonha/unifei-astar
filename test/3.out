Tomou -20 de dano! (quanto mais negativo melhor)
